lib/created_message_queue_handler.ts

```diff
@@ -40,14 +42,17 @@ import {
 import { INotificationEvent } from "./models/notification_event";
 import { ProfileModel } from "./models/profile";
 import { Either, left, right } from "./utils/either";
-import { toNonEmptyString } from "./utils/strings";
+import { NonEmptyString, toNonEmptyString } from "./utils/strings";
 import { Tuple2 } from "./utils/tuples";
 
 // Setup DocumentDB
 
 const COSMOSDB_URI: string = process.env.CUSTOMCONNSTR_COSMOSDB_URI;
 const COSMOSDB_KEY: string = process.env.CUSTOMCONNSTR_COSMOSDB_KEY;
 
+const MESSAGE_CONTAINER_NAME: NonEmptyString =
```

Comment:
what you get from the `env` is a `string`, you must properly convert it to `NonEmptyString` if you want that (either with `isNonEmptyString` or with `toNonEmptyString`)

Reply:
ok that's a more general issue. we don't know if process.env variables are populated till they're used. 

if you use toNonEmptyString() here, when you'll import the module in tests, you'll MUST set the environment variable (even if you use mocks) before importing or the test will throw an exeception calling get().

said that, probably is better here to just use a string instead of NonEmptyString for MESSAGE_CONTAINER_NAME. what about it ?